### ISSUE TITLE
Missing `npm build` in docs

### ISSUE THREAD

**Issue by sienmich** [terraforming-mars/terraforming-mars#2556]
In both docs (<https://docs.google.com/document/d/1r4GlqA6DkrSAtR6MMYmX_nmh6o4igVTqDUUETiJYGt8/edit>, <https://docs.google.com/document/d/1y-QnffzkQtpasBkDAFQwBoqhLmUpVTzRPybtvmbktDQ/edit>) linked in the `README.md` (`Local setup` section) there are missing `npm build` instructions. 

It would be the best if those docs could have been moved to wiki pages.

**Comment by kberg** [comment]
@sienmich I've moved as much as I could to the wiki. If you want to make changes to accommodate this, they belong there. And since we don't really own those docs, I'd just ask that you either ask for edit support or contact those people directly (eg @AbnRgr1978) (Thanks for putting your name on that doc, @AbnRgr1978 !)

**Comment by AbnRgr1978** [comment]
Not positive I understand the email but I happily cede all ownership, editing and credit rights I might have for what I worked on.
 
I was just a tech duffer with a Terraforming Mars passion who did not really understand the steps I was performing to create the local server.  Just happy it worked at the time.
 
You folks are doing a wonderful job with the published project, which I spend time on almost daily.  I believe what I did is outdated and should be taken down as I do not think it is valid anymore.
 
Regardless, do what is best with it for the community with my blessing.
 
AbnRgr1978
 
Aka Abner Rogers
